app/javascript/chat/ChatChannelSettings/InviteForm.jsx

```diff
@@ -26,21 +27,21 @@ const InviteForm = ({
         />
       </div>
       <div>
-        <button
+        <Button
           className="crayons-btn"
```

Comment:
Same comment as above. The CSS class is not necessary when using the design system button.